server.js

```diff
@@ -1,53 +1,53 @@
-var bodyParser = require('body-parser'),
-    config = require(__dirname + '/app/config.js'),
-    express = require('express'),
-    nunjucks = require('express-nunjucks'),
-    path = require('path'),
-    routes = require(__dirname + '/app/routes.js'),
-    app = express(),
-    port = (process.env.PORT || 3000);
+var path = require('path')
+var express = require('express')
+var nunjucks = require('express-nunjucks')
+var routes = require(path.join(__dirname, '/app/routes.js'))
+var app = express()
+var bodyParser = require('body-parser')
+var config = require(path.join(__dirname, '/app/config.js'))
```

Comment:
`path.join`s in `require` calls should not be necessary, they should already be cross-platform compatible.

[I didn't do them on govuk_prototype_kit.](https://github.com/alphagov/govuk_prototype_kit/pull/224/files#diff-78c12f5adc1848d13b1c6f07055d996eR5)